src/test/java/co/rsk/bitcoinj/script/RedeemScriptUtils.java

```diff
@@ -52,7 +55,7 @@ protected static Script createFastBridgeErpRedeemScript(
             .build();
     }
 
```

Comment:
reuse the createFlyoverRedeemScript method declared below.

```
    public static Script createFastBridgeErpRedeemScript(
        List<BtcECKey> defaultRedeemScriptKeys,
        List<BtcECKey> emergencyRedeemScriptKeys,
        Long csvValue,
        byte[] derivationArgumentsHashBytes
    ) {
        ScriptBuilder scriptBuilder = new ScriptBuilder();
        Script erpRedeemScript = createErpRedeemScript(
            defaultRedeemScriptKeys,
            emergencyRedeemScriptKeys,
            csvValue
        );
        return createFlyoverRedeemScript(derivationArgumentsHashBytes, erpRedeemScript);
```

Reply:
also, it would be good to refactor / rename these methods.
for example, `createFastBridgeErpRedeemScript` could be renamed to `createFlyoverErpRedeemScriptFromKeys`

Reply:
These methods will be removed in a new task/PR.